kudu/src/main/java/org/apache/apex/malhar/kudu/AbstractKuduOutputOperator.java

```diff
@@ -339,7 +343,7 @@ private void performCommonProcessing(Operation currentOperation, KuduExecutionCo
       kuduSession.apply(currentOperation);
     } catch (KuduException e) {
       LOG.error("Could not execute operation because " + e.getMessage(), e);
-      throw new RuntimeException(e.getMessage());
+      Throwables.propagateIfPossible(e);
```

Comment:
This isn't propagating all exceptions. See javadoc, you still need to throw non propagated exceptions. Also, why use logging in this case? If you have a custom message, then throw new RuntimeException(msg, e), will be logged by the engine anyways.

Reply:
Ended up reverting to throw RuntimeException(e) because of the reasoning provided here :  https://github.com/google/guava/wiki/Why-we-deprecated-Throwables.propagate

it looks like we need up upgrade guava to a higher version if we need to use the new preferred API ( along with the RuntimeException ) . Since it might disrupt a whole lot of other things, ended up just using the RuntimeException.